File: W2D4-CardViewAndRecyclerView/app/src/main/java/com/example/cardviewandrecyclerview/SecondActivity.java
package com.example.cardviewandrecyclerview;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.app.NavUtils;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;
import android.widget.ImageView;
import android.widget.Toolbar;

import java.util.Objects;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

      getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        CompanyModel companyModel = (CompanyModel) getIntent().getSerializableExtra("company");



        Intent intent = getIntent();
        intent.getStringExtra("name");
        intent.getStringExtra("date");
        intent.getIntExtra("image", 0);


        ImageView imageView = findViewById(R.id.imageViewLogo);

        imageView.setImageResource(companyModel.getCompanyLogo());



    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                NavUtils.navigateUpFromSameTask(this);
                return true;
            default:
                return super.onOptionsItemSelected(item);
        }
    }
}

File: W7D3-ViewAccessApp/app/src/main/java/com/example/viewaccessapp/SecondActivity.kt
package com.example.viewaccessapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.viewaccessapp.databinding.ActivitySecondBinding

class SecondActivity : AppCompatActivity() {

    private lateinit var binding : ActivitySecondBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivitySecondBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.textViewName.text = "Ömer"

        binding.button.setOnClickListener {


        }

    }
}